### PR TITLE
fixes bug

### PR DESCRIPTION
**Pull request by raftx24**

### CODE REVIEW

File: src/routes/api.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Route::namespace('LaravelEnso\ControlPanelApi\App\Http\Controllers')
     ->middleware(['auth:api'])
-    ->prefix('token')->as('token.')
+    ->prefix('api/token')->as('token.')
```
Comment (aocneanu) [inline src/routes/api.php]: I don't think that's ok

Comment (raftx24) [reply src/routes/api.php]: image
these are the addresses that we expect to exists

Comment (aocneanu) [reply src/routes/api.php]: well that is a mistake too
image

Comment (raftx24) [reply src/routes/api.php]: Oh! :(

**Pull request merged**